package.json

```diff
@@ -11,6 +11,7 @@
   "scripts": {
     "test": "./node_modules/karma/bin/karma start",
     "browser-tests": "./node_modules/karma/bin/karma start --browsers Chrome",
+    "karma-watch": "./node_modules/karma/bin/karma start --no-single-run",
```

Comment:
Cool! I'll start using this! Should we create a simple PR for adding this into blip?